shared/components/button/button.svelte

```diff
@@ -0,0 +1,61 @@
+<script lang="ts">
+  import { Button as ButtonPrimitive } from "bits-ui";
+  import { type Events, type Props } from "./index.js";
```

Comment:
We are importing from a index.js, but we have a `index.ts `file in the `button` folder